.php-cs-fixer.php

```diff
@@ -16,10 +16,12 @@
     'array_syntax' => ['syntax' => 'short'],
     'ordered_imports' => ['sort_algorithm' => 'alpha'],
     'no_unused_imports' => true,
-    'braces' => [
-        'position_after_functions_and_oop_constructs' => 'same',
-        'allow_single_line_closure' => true,
-    ],
+    'braces_position' => [
+        'classes_opening_brace' => 'same_line',
```

Comment:
I'm ok with this change, but it might cause a lot of reformatting. There is a mix between sameline and nextline throughout the codebase.

Reply:
I believe this was already happening in the removed part. However, the old name was deprecated because when I ran the formatter without this addition, almost every file was changed. So in theory nothing should actually be changing here other than moving from the old way to new way